streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -849,7 +850,7 @@ object StreamSpec extends ZIOBaseSpec {
                    ).flatMapParSwitch(2)(identity).runDrain.either
           cancelled <- substreamCancelled.get
         } yield assert(cancelled, isTrue) && assert(result, isLeft(equalTo("Ouch")))
-      },
+      } @@ flaky,
```

Comment:
Does this need a ticket?

Reply:
Opened a ticket for it here: https://github.com/zio/zio/issues/2243.